extra_data/cli/make_virtual_cxi.py

```diff
@@ -54,6 +66,7 @@ def main(argv=None):
     )
     ap.add_argument(
         '--fill-value', action='append', nargs=2, metavar=('DS', 'V'),
+        type=parse_number,
```

Comment:
Doesn't this mean it will also try to parse the dataset name as an integer? I know that's unlikely to come up, but it seems like bad design. Maybe do the integer parsing separately after `parse_args`.

It also seems odd that it skips parsing base10 integers.

Reply:
yea, could be `float` or `int`, I thought it'd be ok to let it be cast later (the current behavior)... I'll change that